apps/plasmo/src/components/Calendar/CalendarDay.tsx

```
@@ -0,0 +1,58 @@
+        ${isSelected ? "!plasmo-bg-blue-600" : ""}
+        ${
+          day.isToday && !isSelected
+            ? "plasmo-border-2 plasmo-border-blue-400"
+            : ""
+        }
+        plasmo-flex plasmo-flex-col plasmo-items-center plasmo-justify-center
+        plasmo-transition-all plasmo-duration-200
+        ${day.empty ? "" : "plasmo-cursor-pointer plasmo-hover:bg-gray-700/50"}
+        plasmo-aspect-square
+        relative
+      `}>
```

Biome

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)